prow/config/jobs_test.go

```diff
@@ -162,42 +183,93 @@ func TestCommentBodyMatches(t *testing.T) {
 			"@k8s-bot test this",
 			[]string{},
 		},
+		/* New syntax */
+		{
+			"org/repo",
+			"/ok-to-test",
+			[]string{"gce", "unit"},
+		},
+		{
+			"org/repo",
+			"/ok-to-test\r",
+			[]string{"gce", "unit"},
+		},
+		{
+			"org/repo",
+			"/test",
+			[]string{"gce", "unit", "gke"},
+		},
+		{
+			"org/repo",
+			"/test\r",
+			[]string{"gce", "unit", "gke"},
+		},
+		{
+			"org/repo",
+			"/test unit",
+			[]string{"unit"},
+		},
+		{
+			"org/repo",
+			"/test unit\r",
+			[]string{"unit"},
+		},
+		{
+			"org/repo",
+			"/test federation",
+			[]string{"federation"},
+		},
+		{
+			"org/repo",
+			"/test federation\r",
+			[]string{"federation"},
+		},
+		{
+			"org/repo2",
+			"/test",
+			[]string{"cadveapster"},
+		},
+		{
+			"org/repo3",
+			"/test",
+			[]string{},
+		},
 	}
 	c := &Config{
 		Presubmits: map[string][]Presubmit{
 			"org/repo": {
 				{
 					Name:      "gce",
-					re:        regexp.MustCompile(`@k8s-bot (gce )?test this`),
+					re:        regexp.MustCompile(`(^@k8s-bot (gce )?test this\r?$|^/test( gce)?\r?$)`),
```

Comment:
That's why I'm suggesting you think through what you're actually testing.

**for this test function** what additional coverage are you adding via the old syntax test cases?